services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy

```diff
@@ -50,6 +51,9 @@ class BaseSpecification extends SpringSpecification implements SetupOnce {
     @Autowired
     FlowHelper flowHelper
 
+    @Autowired
+    TopologyHelper topoHelper
```

Comment:
It is not really important but why not topologyHelper? It's not so long name.

Reply:
renamed